packages/web-twig/src/Resources/components/Toast/ToastBar.twig

```diff
@@ -0,0 +1,64 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _closeLabel = props.closeLabel | default('Close') -%}
+{%- set _color = props.color | default('inverted') -%}
+{%- set _hasIcon = props.hasIcon | default(false) -%}
+{%- set _iconName = props.iconName | default(null) -%}
+{%- set _id = props.id | default(null) -%}
+{%- set _isDismissible = props.isDismissible | default(false) -%}
+{%- set _isOpen = props.isOpen ?? true -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'ToastBar' -%}
+{%- set _rootColorClassName = _spiritClassPrefix ~ 'ToastBar--' ~ _color -%}
+{%- set _rootDismissibleClassName = _isDismissible == 'true' ? _spiritClassPrefix ~ 'ToastBar--dismissible' : null -%}
+{%- set _contentClassName = _spiritClassPrefix ~ 'ToastBar__content' -%}
+{%- set _messageClassName = _spiritClassPrefix ~ 'ToastBar__message' -%}
+{%- set _isOpenClassName = _isOpen ? 'is-open' : 'is-hidden' -%}
+
+{# Attributes #}
+{%- set _idAttr = _id ? 'id="' ~ _id | escape('html_attr') ~ '"' : null -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+{%- set _classNames = [ _rootClassName, _rootColorClassName, _rootDismissibleClassName, _isOpenClassName, _styleProps.className ] -%}
+{%- set _iconNameByColor = 'info' -%}
+{% if _color == 'success' %}
+    {%- set _iconNameByColor = 'check-plain' -%}
+{% elseif _color == 'warning' %}
+    {%- set _iconNameByColor = 'warning' -%}
+{% elseif _color == 'danger' %}
+    {%- set _iconNameByColor = 'danger' -%}
+{% endif %}
+{%- set _iconNameValue = _iconName | default(_iconNameByColor) -%}
+{%- set _mainPropsWithoutId = props | filter((value, prop) => prop is not same as('id')) -%}
```

Comment:
I think in the latest components we avoided this name because then you have to rename it when you add another filtered prop